packages/css/lib/index.js

```diff
@@ -1,11 +1,15 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.minify = void 0;
+exports.minifySync = exports.minify = void 0;
 const binding = require("./binding");
```

Comment:
Please remove this file

Reply:
Remove binding.js?

Reply:
No, `index.js`

Reply:
Does the current CI automatically generate index.js when npm publish?

Reply:
Yes

Reply:
Done